payload_creator/serverless.yml

```diff
@@ -57,13 +57,27 @@ resources:
         BucketName : ${self:provider.environment.POC_PPTX_BUCKET}
         PublicAccessBlockConfiguration:
           BlockPublicAcls: true
-          BlockPublicPolicy: true
+          BlockPublicPolicy: false
           IgnorePublicAcls: true
-          RestrictPublicBuckets: true
+          RestrictPublicBuckets: false
         BucketEncryption:
           ServerSideEncryptionConfiguration:
             - ServerSideEncryptionByDefault:
                 SSEAlgorithm: AES256
+
+    BucketPolicy:
+      Type: AWS::S3::BucketPolicy
+      Properties:
+        Bucket:
+          Ref: 'PocDocx'
+        PolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+          - Sid: 'PublicRead' 
+            Effect: Allow
+            Principal: '*'
```

Comment:
Give our account id as the principle because now anyone in the world can access this object.